tests/installation/releasenotes_offline.pm

```diff
@@ -21,11 +21,12 @@ sub run {
     assert_screen('release-notes-button');
     send_key('ctrl-shift-alt-x');
     assert_screen('yast-xterm');
-    my $src = check_var('SCC_REGISTER', 'installation') ? "RPM" : "URL";
+    # This is releasenotes_offline so we expect release notes from RPM
```

Comment:
this comment doesn't provide any valuable info which is not already obvious from the code itself, please delete it.